### PR TITLE
ERM-999: Add record/row identifiers into Local KB Admin log display

### PR DESCRIPTION
**Pull request by peterboehm**

### CODE REVIEW

File: lib/LogsList/LogsList.js [hunk 1]
```diff
@@ -5,40 +5,53 @@ import { MultiColumnList, NoValue, Spinner } from '@folio/stripes/components';
...
+            message: PropTypes.string,
+        })),
+        onNeedMoreLogs: PropTypes.func.isRequired,
+        type: PropTypes.string.isRequired,
+    };
 
-  render() {
-    const { job, logs, onNeedMoreLogs, type } = this.props;
+    resultsFormatter = {
+        recordDescriptor: ({ additionalInfo }) => {
```
Comment (aditya-matukumalli) [inline lib/LogsList/LogsList.js:19]: ``
```suggestion
        recordDescriptor: ({ additionalInfo = {} }) => {
```

Comment (aditya-matukumalli) [inline lib/LogsList/LogsList.js:19]:
```suggestion
        recordDescriptor: ({ additionalInfo = {} }) => {
          const { packageSource, packageSource, rowNumber, recordNumber } = additionalInfo;
```

File: lib/LogsList/LogsList.js [hunk 2]
```diff
@@ -5,40 +5,53 @@ import { MultiColumnList, NoValue, Spinner } from '@folio/stripes/components';
...
+            return (
+                <div>
+                    {additionalInfo.packageSource ? <FormattedMessage id="stripes-erm-components.packageSource" tagName="div" values={{ source: additionalInfo.packageSource }} /> : null}
+                    {additionalInfo.packageReference ? <FormattedMessage id="stripes-erm-components.packageReference" tagName="div" values={{ ref: additionalInfo.packageReference }} /> : null}
+                    {additionalInfo.rowNumber ? <FormattedMessage id="stripes-erm-components.rowNumber" tagName="div" values={{ rownumber: additionalInfo.rowNumber }} /> : null}
+                    {additionalInfo.recordNumber ? <FormattedMessage id="stripes-erm-components.recordNumber" tagName="div" values={{ recordnumber: additionalInfo.recordNumber }} /> : null}
+                </div>
```
Comment (aditya-matukumalli) [inline lib/LogsList/LogsList.js:26]: With the above requested change the `additionalInfo.packageSource ` could just be replaced with `packageSource`. Similarly with  `packageReference`, `rowNumber` and `recordNumber`. Its more cleaner that way as we dont need to fetch the property from `additionalInfo` everytime.

File: translations/stripes-erm-components/en.json [hunk 3]
```diff
@@ -15,7 +15,11 @@
   "errorLogNo": "No errors to report.",
   "columns.errorLogMessage": "Error",
   "columns.infoLogMessage": "Message",
-  "columns.recordNumber": "Record number",
+  "columns.recordDescriptor": "Record descriptor",
```
Comment (aditya-matukumalli) [inline translations/stripes-erm-components/en.json:18]: Its best to leave an empty line between lines 18 and 19 just to differentiate the related translations

File: translations/stripes-erm-components/en_US.json [hunk 4]
```diff
@@ -197,7 +197,11 @@
     "errorLogNo": "No errors to report.",
     "columns.errorLogMessage": "Error",
     "columns.infoLogMessage": "Message",
-    "columns.recordNumber": "Record number",
+    "columns.recordDescriptor": "Record descriptor",
+    "packageSource": "Package source: {source}",
+    "packageReference": "Package reference: {ref}",
+    "rowNumber": "Row number: {rownumber}",
+    "recordNumber": "Record number: {recordnumber}",
```
Comment (aditya-matukumalli) [inline translations/stripes-erm-components/en_US.json:204]: Only translations that are already present have to be edited in the `en_US.json` file. New translations `shouldn't` be added here and adding them only to `en.json` should suffice.

**Review by aditya-matukumalli — CHANGES_REQUESTED** [review]
Looks good to me. Some minor tweaks and linitng errors need to be fixed.

**Comment by peterboehm** [comment]
@aditya-matukumalli Thanks for the suggestions! I updated the code accordingly

**Review by aditya-matukumalli — APPROVED** [review]

**Pull request merged**